guides/common/modules/con_prerequisites-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,45 @@
+[id="prerequisites-for-{project-context}-on-aws"]
+= Prerequisites for {Project} on AWS
+
+Installing and running {ProjectServer} and {SmartProxyServers} on Amazon Web Services (AWS) has additional requirements to your environment.
```

Comment:
You might not even need this sentence.

Reply:
Or perhaps be more specific instead:
```suggestion
In additional to the standard prerequisites for installing and running {ProjectServer} and {SmartProxyServers}listed in <links to prereqs>, ensure to also meet the following prerequisites specific to deploying on Amazon Web Services (AWS):
```

This is way longer but also much more specific than the rather vague wording about "additional requirements".